dags/nocp/scripts/run_ocm_benchmark.sh

```diff
@@ -0,0 +1,80 @@
+#!/bin/bash
+
+export UUID=$(uuidgen | head -c8)-$AIRFLOW_CTX_TASK_ID-$(date '+%Y%m%d')
+echo "############################################"
+echo "# Benchmark UUID: ${UUID}"
+echo "############################################"
+
+env >> /tmp/environment.txt
+echo $UUID
+
+run_ocm_benchmark(){
+    echo "OCM Benchmark will be began.."
```

Comment:
Small typo should be `begin`, it's OK to skip it if no more changes are done.

Reply:
done